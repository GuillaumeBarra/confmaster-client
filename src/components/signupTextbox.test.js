import React from 'react'
import {render} from '@testing-library/react'
import SignupTextbox from './signupTextbox'
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<SignupTextbox />)
    
    expect(asFragment(<SignupTextbox />)).toMatchSnapshot()
   })