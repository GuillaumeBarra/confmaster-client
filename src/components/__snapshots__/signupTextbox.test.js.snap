// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-dkPtRN beWxGE"
  >
    <div
      class="sc-gsDKAQ gtLMAp"
    />
    <div
      class="MuiInputBase-root sc-bdvvtL cPIRI"
      sx="[object Object]"
    >
      <input
        class="MuiInputBase-input"
        type="text"
        value=""
      />
    </div>
  </div>
</DocumentFragment>
`;
